service/internal/logger/audit/policy.go

```diff
@@ -0,0 +1,84 @@
+package audit
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+)
+
+// Object Types for Policies
+const (
+	ObjectTypeSubjectMapping                   = "subject_mapping"
+	ObjectTypeResourceMapping                  = "resource_mapping"
+	ObjectTypeAttributeDefinition              = "attribute_definition"
+	ObjectTypeAttributeValue                   = "attribute_value"
+	ObjectTypeNamespace                        = "namespace"
+	ObjectTypeConditionSet                     = "condition_set"
+	ObjectTypeKasRegistry                      = "kas_registry"
+	ObjectTypeKasAttributeDefinitionAssignment = "kas_attribute_definition_assignment"
+	ObjectTypeKasAttributeValueAssignment      = "kas_attribute_value_assignment"
+)
```

Comment:
golang doesn't have the concept of enums but you can still sort of represent them if you think it might be beneficial here and also for action types.

Here are a couple of examples
- https://www.sohamkamani.com/golang/enums/
- https://builtin.com/software-engineering-perspectives/golang-enum

Reply:
@strantalis made those updates now and put them inside `audit/constants.go`